.github/CODEOWNERS

```
@@ -0,0 +1,13 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       olle.jonsson@gmail.com
+*       rod@foveacentral.com
```

Since it support usernames, we can productively use that here:

```suggestion
*       @olleolleolle
*       @ivanoblomov 
```

---

I read the docs https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection